### ISSUE TITLE
AbpJsonOptions.DefaultDateTimeFormat = "yyyy-MM-dd HH:mm:ss"  json Deserialize failed

### ISSUE THREAD

**Issue by zhangqx1** [abpframework/abp#6081]
i will get json string format to "yyyy-MM-dd HH:mm:ss" ,but when i set AbpJsonOptions ,  frontend failed to sends the POST data, . The response body says:

```
The supplied value is invalid.
```
The value of the invalid lastUpdated property being sent is:
```
2020-11-06
```

MyprojectHttpApiHostModule
```
context.Services.Configure<AbpJsonOptions>(options =>
 {
     options.DefaultDateTimeFormat = "yyyy-MM-dd HH:mm:ss";
 });
```

TestController
```
[Route("api/test")]
    public class TestControlller : AbpController
    {
        [HttpGet]
        public TestModel Get()
        {
            return new TestModel()
            {
                LastUpdated = DateTime.Now
            };
        }
        [HttpPost]
        public void Post(TestModel input)
        {
            //failed
        }
    }
```

**Comment by maliming** [comment]
You can try to send date time as `2020-11-06 00:00:00`

**Comment by hexianggui** [comment]
> You can try to send date time as `2020-11-06 00:00:00`

ERROR:Volo.Abp.Validation.AbpValidationException: ModelState is not valid!  When I send "yyyy-MM-dd HH:mm:ss" date format

**Comment by fanslead** [comment]
I have the same problem. How can fix it?

**Comment by tranthebao** [comment]
How can fix it for net 6.0
